src/services/favoritesService.js

```diff
@@ -23,3 +23,12 @@ export const addToFavorite = async (data) => {
     // res.status(StatusCodes.BAD_REQUEST).json({ message: err.message });
   }
 };
+
+export const findAllFavorites = async () => {
+  try {
+    const favorites = await favoriteModel.find();
```

Comment:
Once the house model is ready and this endpoint returns some basic info about houses, the data in the response should probably also be paginated.